draft-ietf-teep-protocol.md

```diff
@@ -1186,3 +1186,179 @@ ta-uuid = 16
 ta-manifest-sequence-number = 17
 have-binary = 18
 ~~~~
+
+# D. Examples of Diagnostic Notation and Binary Representation {#examples}
+{: numbered='no'}
+
+### Some assumptions in examples
+
+- TAM supports 2 versions: 0, 1
+- OCSP stapling data = h'010203'
+- TEEP Device has 2 TAs
+  - TA-ID: 0x010203, 0x040506
```

Comment:
For consistency with PR #53 the TA IDs should be 16 bytes long

Reply:
I revised the TA ID to be 16 bytes https://github.com/ietf-teep/teep-protocol/pull/65/commits/b0dd2fcdc2418fd38d5a996d2b61ff4c103016c7